tests/testthat/test-interact.R

```diff
@@ -127,3 +127,17 @@ test_that("run_safely_loudly works for result", {
 
    log_remove()
 })
+
+test_that("run_file uses a child of the global environment for execution", {
+   expect_equal("<environment: R_GlobalEnv>\n",
+                capture_message(logrx:::run_file(test_path("ref", "run_file_test_parent_env.R")))$message)
+
+})
+
+test_that("run_file uses the global env if set using log.rx.exec.env", {
```

Comment:
@kodesiba and @bms63 I'd suggest that we make this test a separate environment than global, because the if the log.rx.exec.env option wasn't set, it would produce the same result.

Reply:
happy to test it somewhere else but this check slightly differs since it's checking the current env of execution and the one above is checking the parent.